test/testvm.py

```diff
@@ -309,7 +309,7 @@ def upload(self, sources, dest):
         if self.verbose:
             subprocess.check_call(cmd)
         else:
-            with stdchannel_redirected(sys.stderr, os.devnull):
+            with stdchannel_redirected(sys.stdout, os.devnull), stdchannel_redirected(sys.stderr, os.devnull):
```

Comment:
We shouldn't be hiding stderr from scripts. They contain useful information about how a command fails. If a particular caller wants to silence stderr, they should do so with a /dev/null reroute.